docs/v3/get-started/quickstart.mdx

```diff
@@ -1,179 +1,125 @@
+        return results
+
+
+    if __name__ == "__main__":
+        main()
+
+    ```
+
+    This looks like the ordinary Python you'd write, except there is 
+        - a `@flow` decorator on the script’s entrypoint.
+        - a `@task` decorator on each function called within the flow.
+
+    When you run this `flow`, Prefect dynamically creates a graph of each `task` and the state of their upstream dependencies. This allows Prefect to execute each `task` in the right order and, in the case of failure, to recover the state of your workflow and resume from its point of failure. 
+    
+    Let's run this code locally on your computer. To do that, run the following command in your terminal.
+    ```bash
+    git clone https://github.com/PrefectHQ/quickstart && cd quickstart
+    uv run 01_getting_started.py
+    ```
+    If all went well, you'll see a link to see its execution graph in the Prefect UI followed by a flurry of state changes and logs as Prefect dynamically discovers and executes your workflow.
+
+    ```bash {11} [expandable]
+    00:30:53.633 | INFO    | Flow run 'airborne-ringtail' - Beginning flow run 'airborne-ringtail' for flow 'main'
+    00:30:53.638 | INFO    | Flow run 'airborne-ringtail' - View at https://app.prefect.cloud/account/...
+    00:30:53.685 | INFO    | Task run 'get_customer_ids-136' - Finished in state Completed()
+    00:30:54.512 | INFO    | Task run 'process_customer-d9b' - Finished in state Completed()
+    00:30:54.518 | INFO    | Task run 'process_customer-113' - Finished in state Completed()
+    00:30:54.519 | INFO    | Task run 'process_customer-1c6' - Finished in state Completed()
+    00:30:54.519 | INFO    | Task run 'process_customer-30d' - Finished in state Completed()
+    00:30:54.520 | INFO    | Task run 'process_customer-eaa' - Finished in state Completed()
+    00:30:54.523 | INFO    | Task run 'process_customer-b2b' - Finished in state Completed()
+    00:30:54.523 | INFO    | Task run 'process_customer-90a' - Finished in state Completed()
+    ...Ommitted for brevity in this tutorial
+    00:30:54.553 | INFO    | Task run 'process_customer-4af' - Finished in state Completed()
+    00:30:54.555 | INFO    | Task run 'process_customer-e66' - Finished in state Completed()
+    00:30:54.556 | INFO    | Task run 'process_customer-e7e' - Finished in state Completed()
+    00:30:54.556 | INFO    | Task run 'process_customer-5bd' - Finished in state Completed()
+    00:30:54.557 | INFO    | Task run 'process_customer-05d' - Finished in state Completed()
+    00:30:54.558 | INFO    | Task run 'process_customer-b60' - Finished in state Completed()
+    00:30:54.701 | INFO    | Flow run 'airborne-ringtail' - Finished in state Completed('All states completed.')
+    ```
+    </Step>
+    <Step title="Where should it run?">
+    At this point, you've successfully run a Prefect `flow` locally. Let's get it running off of your machine! Again, for simplicity, 
```

Comment:
Because the runs look so close (local vs remote), I wonder if theres any stronger language we can use to re-iterate that they've deployed something and can run code from a repo entirely remotely. I don't have a suggestion here, just a vibe.

![Screenshot 2025-04-04 at 11 38 39 AM](https://github.com/user-attachments/assets/016aff7c-1c59-4f7a-92b5-3ad95d1f72ae)